.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+*Please edit the list to your PR scope, then check off the boxes as you go!*
```

Comment:
*Start with a description of this PR. Then edit the list below to the items that make sense for your PR scope, and check off the boxes as you go!*